recipes/bottle/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "bottle" %}
+{% set version = "0.12.9" %}
+{% set sha256 = "45285ad084ca054d821ceef8bd95462efd38e13ecbef13a82c22b6472a6f4b2d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/bottlepy/bottle/archive/{{ version  }}.tar.gz
```

Comment:
We had this discussion awhile back. Some people wanted to build directly from source. Other preferred going with a package index. After a fair bit of discussion, we decided to let both continue. The only caveats are that we must download some kind of compressed file (no VCS clones) and if it comes from source it must be the original source (no forks).